app/screens/home/channel_list/servers/servers_list/server_item/options/index.tsx

```diff
@@ -46,6 +52,33 @@ const ServerOptions = ({progress, server}: Props) => {
         console.log('ON Remove');
     }, [server]);
 
+    let sessionComponent;
+    if (isLoggedIn) {
+        sessionComponent = (
+            <Option
+                color='#F58B00'
```

Comment:
Shouldn't this be a value relative to the theme? (to work well with any theme)

Reply:
this was requested to be HARDCODED by @Rina-dsg as the `theme` value did not work well with all stock themes.

Reply:
oh and it was also approved by @matthewbirtch

Reply:
I am not a big fan of harcoded values for colors. They get broken really fast with custom themes, which imply problems in accessibility (see color blindness themes). I would be happy to keep it like this for the short term, but would love to see at least a ticket to revise this.